AEPMessaging/Sources/LiveActivity/Messaging+LiveActivityPublicAPI.swift

```diff
@@ -232,26 +231,31 @@ public extension Messaging {
     /// - Parameter activity: The newly started `Activity` instance. The activity must conform to ``LiveActivityAttributes``.
     private static func dispatchStartEvent<T: LiveActivityAttributes>(activity: Activity<T>) {
         let attributeTypeName = T.attributeTypeName
-        let liveActivityID = activity.attributes.liveActivityData.liveActivityID ?? MessagingConstants.Event.Data.Value.UNAVAILABLE
+        let liveActivityIdentifierData = activity.attributes.liveActivityIdentifierData
```

Comment:
is one of channel id or live activity id always available?  if so, do we still need the constant `MessagingConstants.Event.Data.Value.UNAVAILABLE`?